test/buy-tshirt.spec.ts

```diff
@@ -23,19 +23,23 @@ describe('Buy a t-shirt', () => {
   const summaryStepPage: SummaryStepPage = new SummaryStepPage();
 
   it('then should be bought a t-shirt', async () => {
-    await browser.get('http://automationpractice.com/');
-    await menuContentPage.goToTShirtMenu();
-    await addressStepPage.addToCar();
-    await bankPaymentPage.proceedToCheckout();
-    await orderSummaryPage.proceedToCheckout();
-    await paymentStepPage.fillEmailPassword();
-    await paymentStepPage.registerEmail();
-    await productAddedModalPage.proceedToCheckout();
-    await shippinStepPage.selectCheckBoxAgreeTerms();
-    await productListPage.proceedToCheckout();
-    await signInStepPage.payByBankWire();
-    await summaryStepPage.confirmOrder();
-    await expect(summaryStepPage.mensajeText())
+    describe('when open Products Page', () => {
```

Comment:
este cambio no se debe hacer ya que cambiaste el expect por fuera de un it, el it se usa para señalar donde se hacen los asserts, por que el cambio?

Reply:
ya se resolvio el inconveniente